packages/spec-dashboard/src/apis.ts

```diff
@@ -8,30 +8,18 @@ const storageAccountName = "typespec";
 
 export type GeneratorNames =
   | "@typespec/http-client-python"
-  | "@azure-tools/typespec-python"
-  | "@azure-tools/typespec-go"
-  | "@azure-tools/typespec-csharp"
   | "@typespec/http-client-csharp"
   | "@azure-tools/typespec-ts-rlc"
   | "@azure-tools/typespec-ts-modular"
-  | "@azure-tools/typespec-java"
   | "@typespec/http-client-java"
-  | "@azure-tools/typespec-cpp"
-  | "@azure-tools/typespec-rust"
   | "test";
 const query = new URLSearchParams(window.location.search);
 const generatorNames: GeneratorNames[] = [
   "@typespec/http-client-python",
-  "@azure-tools/typespec-python",
-  "@azure-tools/typespec-go",
-  "@azure-tools/typespec-csharp",
   "@typespec/http-client-csharp",
   "@azure-tools/typespec-ts-rlc",
   "@azure-tools/typespec-ts-modular",
```

Comment:
These can't be removed too?

Reply:
These are special case and they do generate the standard.json. I am not sure why. I have asked them the reason. Based on their response, I will remove them.